### PR TITLE
Bugfix/0.0.4 defects

### PR DESCRIPTION
**Pull request by iandesj**
## Working PR to show diff of all the defects/fixes I'm making before release 0.0.5

### CODE REVIEW

File: src/app/pages/wallet-detail/wallet-detail-page.component.ts [hunk 1]
```diff
@@ -83,6 +83,9 @@ export class WalletDetailPageComponent implements OnDestroy, OnInit {
           }
           await this.transactionsPersistenceService.db.bulkDocs(transactions);
         }
+      }
+      if (i % 100) {
+        this.lastBlockNumberSynced = i;
         localStorage.setItem(`lastBlock_${this.wallet.address}`, i.toString());
```
Comment (iandesj) [inline src/app/pages/wallet-detail/wallet-detail-page.component.ts]: todo: keep in pouch

**Comment by seanjnkns** [comment]
Only issue I'm seeing is this:

1) With an already synced block chain from a prior release, unless you clear the cache, it'll never sync

Otherwise, aforementioned checked off issues mentioned in #41 appear resolved.

**Comment by seanjnkns** [comment]
Another issue is that after a wallet syncs 100%, upon later revisiting that wallet, there's no indication if it's 100% synced or not.  Need an indicator that it's out of sync and catching up.

**Comment by iandesj** [comment]
Merging for 0.0.5 builds.

**Pull request merged**